doc/mgr/influx.rst

```diff
@@ -0,0 +1,185 @@
+=============
+Influx Plugin 
+=============
+
+The influx plugin continuously collects and sends time series data to an influxdb database. Users have the option to specify what type of stats they want to collect. 
+Some default counters are already set. However, users will have the option to choose some additional counters to collect. 
+
+-------------
+Configuration 
+-------------
+
+In order for this module to work, the following configuration should be created ``/etc/ceph/influx.conf``.
+
+^^^^^^^^
+Required 
+^^^^^^^^
+
+The configurations must include the following under the header ``[influx]``.
```

Comment:
@mhdo2 Sentence looks incorrect.
either it should be something.. `Following configuration must be included under header`.